etc/skeleton/driver/main.go.tpl

```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"gopkg.in/bblfsh/sdk.v1/protocol/driver"
 
-	"github.com/bblfsh/{{.Manifest.Language}}-driver/driver/normalizer"
+	"gopkg.in/bblfsh/sdk.v1/etc/skeleton/driver/normalizer" //REPLACE:"github.com/bblfsh/{{.Manifest.Language}}-driver/driver/normalizer"
```

Comment:
This change shouldn't be here, since it refers to driver's module, not sdk's module.